packages/react-components/react-virtualizer/src/hooks/useImperativeScrolling.types.ts

```diff
@@ -0,0 +1,21 @@
+import { MutableRefObject, RefObject } from 'react';
```

Comment:
It looks like there are a lot of type values like that imported without the `type` keyword. Does this repo need it enforced?

Reply:
This Eslint plugin can auto sort imports, including type imports: [eslint-plugin-simple-import-sort
](https://github.com/lydell/eslint-plugin-simple-import-sort#readme). It can works with both `import type { X }` and `import { X, type Z }` syntax.

Reply:
… and then this rule: [consistent-type-imports](https://typescript-eslint.io/rules/consistent-type-imports/) from typescript-eslint can detect when an import is used only as a type and auto fix.